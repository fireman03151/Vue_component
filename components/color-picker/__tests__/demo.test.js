import demoTest from '../../../tests/shared/demoTest';

demoTest('color-picker');
