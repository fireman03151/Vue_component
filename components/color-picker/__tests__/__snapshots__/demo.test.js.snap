// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./antdv-demo/docs/color-picker/demo/alpha.md correctly 1`] = `<a-color-picker alpha="true"></a-color-picker>`;

exports[`renders ./antdv-demo/docs/color-picker/demo/basic.md correctly 1`] = `
<div class="ant-row">
  <div class="ant-col ant-col-12">
    有默认值
    <a-colorpicker></a-colorpicker>
  </div>
  <div class="ant-col ant-col-12">
    无默认值
    <a-colorpicker></a-colorpicker>
  </div>
</div>
`;

exports[`renders ./antdv-demo/docs/color-picker/demo/colors.md correctly 1`] = `<a-color-picker config="[object Object]"></a-color-picker>`;

exports[`renders ./antdv-demo/docs/color-picker/demo/hue.md correctly 1`] = `<a-color-picker></a-color-picker>`;

exports[`renders ./antdv-demo/docs/color-picker/demo/size.md correctly 1`] = `
<div>
  <div class="ant-row">
    <div class="ant-col ant-col-8">
      <a-colorpicker size="large"></a-colorpicker>
    </div>
    <div class="ant-col ant-col-8">
      <a-colorpicker></a-colorpicker>
    </div>
    <div class="ant-col ant-col-8">
      <a-colorpicker size="small"></a-colorpicker>
    </div>
  </div>
</div>
`;
